Corona-Warn-App/src/main/res/values-de/family_tests_strings.xml

```diff
@@ -69,7 +69,9 @@
     <!-- XHED: family tests list: Deletion alert window: body all tests -->
     <string name="family_tests_list_deletion_alert_body_all">"Die Testergebnisse werden in den Papierkorb verschoben. Den Papierkorb finden Sie auf der „Mehr“-Kachel der Status-Registerkarte. Dort können Sie diese endgültig löschen oder wiederherstellen. Nach 30 Tagen werden die Testergebnisse automatisch gelöscht."</string>
```

Comment:
Instead of "Den Papierkorb finden Sie auf der „Mehr“-Kachel der Status-Registerkarte." it should be "Den Papierkorb finden Sie auf der Status-Registerkarte unter den 3 Punkten oben rechts."